build/images/scripts/PrepareNode.ps1

```diff
@@ -0,0 +1,120 @@
+<#
```

Comment:
I don't think this folder is the best location for this script, as it is something that the user has to run manually. Maybe under `/hack/windows`?

Reply:
Same comment for `install_ovs_windows.ps1` and `uninstall_ovs_windows.ps1`

Reply:
Sure, I think place them under `/hack/windows` is better. Done.